src/test/java/org/mule/extension/db/integration/model/MySqlTestDatabase.java

```diff
@@ -150,6 +150,24 @@ public void createStoredProcedureAddOne(DataSource dataSource) throws SQLExcepti
     createStoredProcedure(dataSource, sql);
   }
 
+  @Override
+  public void createStoredProcedureAddOneNoSchema(DataSource dataSource) throws SQLException {
+    executeDdl(dataSource, "DROP PROCEDURE IF EXISTS mathFunction.addOne");
+    executeDdl(dataSource, "DROP PROCEDURE IF EXISTS addOne");
+
+    String sql = "CREATE PROCEDURE mathFunction.addOne(INOUT num INT)\n" +
```

Comment:
Why are you dropping and creating the `mathFunction.addOne` here. There is another method that creates it

Reply:
Also, in MySQL, schema and database are the same, so there is always a schema